src/Shared/Configuration/Files/Web.cs

```diff
@@ -17,8 +18,12 @@ public class WebConfFile : ConfFile
 		public void Load(string filePath)
 		{
 			this.Include(filePath);
-
-			this.PhpCgiFilePath = this.GetString("php_cgi_bin", Path.Combine("user", "tools", "php", "php-cgi.exe"));
+            if (Environment.OSVersion.Platform == PlatformID.Win32NT)
```

Comment:
This won't really change anything, because the configuration value is read from the conf file regardless of the operating system. The only way you will get the usr bin path from this is if the conf option is missing.